src/main/java/com/vonage/quickstart/voice/MuteCall.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright  2020 Vonage
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package com.vonage.quickstart.voice;
+
+import com.vonage.client.VonageClient;
+import com.vonage.client.voice.Call;
+import com.vonage.client.voice.CallEvent;
+import com.vonage.client.voice.ModifyCallAction;
+import com.vonage.quickstart.Util;
+
+public class MuteCall {
+    public static void main(String... args) throws Exception {
+        Util.configureLogging();
+
+        String VONAGE_APPLICATION_ID = Util.envVar("VONAGE_APPLICATION_ID");
+        String VONAGE_PRIVATE_KEY_PATH = Util.envVar("VONAGE_PRIVATE_KEY_PATH");
+
+        VonageClient client = VonageClient.builder()
+                .applicationId(VONAGE_APPLICATION_ID)
+                .privateKeyPath(VONAGE_PRIVATE_KEY_PATH)
+                .build();
+
+        final String VONAGE_NUMBER = Util.envVar("VONAGE_NUMBER");
+        final String TO_NUMBER = Util.envVar("TO_NUMBER");
+        /*
+        Establish a call for testing purposes.
+         */
+        CallEvent call = client.getVoiceClient().createCall(new Call(
+                TO_NUMBER,
+                VONAGE_NUMBER,
+                "https://gist.githubusercontent.com/cr0wst/9417cac4c0d9004805a04aed403ae94a/raw/b95e3cd5126587d25986e0bf832eb33a7538394d/tts_long.json"
```

Comment:
Same with this one - might be good to move those directly to this repo.

Reply:
fixed